streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -49,6 +48,7 @@
  * {@link KafkaStreams} instance that will then {@link KafkaStreams#start() begin consuming, processing, and producing
  * records}.
  */
+@SuppressWarnings("unchecked")
```

Comment:
This is a meta comment: instead of adding the annotation on the class level, could we add it on each function that has warnings so we know which ones are the culprits so in the future we may look to resolve them?